app/javascript/components/explore/ExploreDisplayTabs.jsx

```diff
@@ -432,10 +434,13 @@ export default function ExploreDisplayTabs({
               />
             }
             { !showViewOptionsControls &&
-              <button className="action view-options-toggle view-options-toggle-on"
+              <button className={showDifferentialExpressionPanel ?
+                "action view-options-toggle view-options-toggle-on" :
+                "action minified-options view-options-toggle view-options-toggle-on"
```

Comment:
Easier visual diff:

```suggestion
                "action view-options-toggle view-options-toggle-on" :
                "action view-options-toggle view-options-toggle-on minified-options"
```

For even better maintainability, I think this could be something conciser like:

```
className={
 'action view-options-toggle view-options-toggle-on ' +
 showDifferentialExpressionPanel ? '' : 'minified-options'
}
```

But I haven't actually tried that locally.

Reply:
[4e68b70](https://github.com/broadinstitute/single_cell_portal_core/pull/1794/commits/4e68b70238f93feffbfb92dfb44e181f08d5ee89)